### PR TITLE
Implmented ValidationUtil Email Validation

### LINKED ISSUE #252: No Email validator present in the Sign Up Details page.
**Summary:**There is no validation check for correct email while filling up details.

**Steps to reproduce:**
Go to Sign up.
Proceed.
enter details.

**Expected behavior:**
To have a validation check on  the email,

**Observed behavior:**

What did you see instead?  Describe your issue in detail here.

**Device and Android version:**

What make and model device (e.g., Samsung Galaxy S3) did you encounter this on?  What Android
version (e.g., Android 4.0 Ice Cream Sandwich or Android 6.0 Marshmallow) are you running?  Is it
 the stock
version from the manufacturer or a custom ROM?

**Screenshots:**

### PR DESCRIPTION
**Pull request by jddeep**
Fixes #252 

Please Add Screenshots If any UI changes.

Please Add Summary of what changes you have made.(Optional)

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by jddeep** [comment]
@ankurs287  I implemented the already present Email validator. Now the PR can be reviewed and merged. Thanks.

**Comment by jddeep** [comment]
@luckyman20 @ankurs287 @codepoet2017390 The Travis build error has been fixed. Now the PR can be reviewed and merged. Thanks.

**Comment by jddeep** [comment]
@luckyman20 @ankurs287 Kindly look into the PR. Please review it.

File: core/build.gradle [hunk 1]
```diff
@@ -56,4 +56,4 @@ dependencies {
     implementation 'io.reactivex:rxjava:1.1.4'
 
     testImplementation 'junit:junit:4.12'
-}
+}
```
Comment (luckyman20) [inline core/build.gradle]: Remove this change.

File: mifospay/build.gradle [hunk 2]
```diff
@@ -84,4 +84,4 @@ dependencies {
     implementation project(':core')
 }
 
-apply plugin: 'com.google.gms.google-services'
+apply plugin: 'com.google.gms.google-services'
```
Comment (luckyman20) [inline mifospay/build.gradle]: This too.

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/auth/ui/LoginActivity.java [hunk 3]
```diff
@@ -181,4 +181,4 @@ public void onComplete(@NonNull Task<Void> task) {
                     });
         }
     }
-}
+}
```
Comment (luckyman20) [inline mifospay/src/main/java/org/mifos/mobilewallet/mifospay/auth/ui/LoginActivity.java]: This too.

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/registration/ui/SignupActivity.java [hunk 4]
```diff
@@ -219,6 +220,12 @@ public void onNextClicked() {
             return;
         }
 
+        if (!ValidateUtil.isValidEmail(email)) {
+            Toaster.showToast(this, "Please enter correct Email Address.");
```
Comment (luckyman20) [inline mifospay/src/main/java/org/mifos/mobilewallet/mifospay/registration/ui/SignupActivity.java]: Use Snackbar instead of Toaster.

Comment (jddeep) [reply mifospay/src/main/java/org/mifos/mobilewallet/mifospay/registration/ui/SignupActivity.java]: Resolved @luckyman20

File: core/build.gradle (hunk 1 above)
Comment (jddeep) [reply core/build.gradle]: I can't understand what is the change here. The closing braces are at the same position and line number as earlier.

File: mifospay/src/main/res/layout/activity_signup.xml [hunk 5]
```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <RelativeLayout
+    android:id="@+id/Signup_parent"
```
Comment (luckyman20) [inline mifospay/src/main/res/layout/activity_signup.xml]:
```suggestion
    android:id="@+id/signup_parent"
```

File: core/build.gradle (hunk 1 above)
Comment (jddeep) [reply core/build.gradle]: @luckyman20 i unstaged a particular file using ` git checkout HEAD^ path/of/file `

Comment (jddeep) [reply core/build.gradle]: But still after that these changes seem to remain.

**Comment by luckyman20** [comment]
@jddeep Since this PR has merge conflicts too, close this one and make a new.

**Comment by luckyman20** [comment]
@jddeep Please let us know if you're going to work on this PR. Will wait for 3 days otherwise it would be closed.

**Comment by s-ayush2903** [comment]
@luckyman20 I'm ready to take up the corresponding issue to fix as soon as this PR gets inactive

**Comment by jddeep** [comment]
Hey @luckyman20, have lost track of this PR completely. Am closing this. Contributors are free to take up this. Thanks.

**Pull request closed without merging**